src/Interbase-1.0.xml

```diff
@@ -531,15 +519,17 @@
              of the Covered Code under Your choice of the NPL or the alternative licenses, if any,
              specified by the Initial Developer in the file described in Exhibit A.</p>
             <p>EXHIBIT A - InterBase Public License.</p>
-            <p>``The contents of this file are subject to the Interbase Public License Version 1.0 (the
+            <standardLicenseHeader>
+            <p><optional>"</optional>The contents of this file are subject to the Interbase Public License Version 1.0 (the
```

Comment:
I standardized the LaTeX-ish double-backtick to `"`.  This licence remains wildly inconsistent about quotation characters, but that was the only instance of double backticks.